docs/source/developer_guide/logging.md

```diff
@@ -66,7 +66,7 @@ The log output will be color coded, legend as follows:
 | ERROR     | Red   |
 | CRITICAL  | Magenta |
 
-#### Setting the Log Level in Elyra
+##### Setting the Log Level in Elyra
```

Comment:
The `in Elyra` clause was dropped at the top, we should probably do the same here.

Reply:
Agreed. Will make this change.